### ISSUE TITLE
color in preview

### ISSUE THREAD

**Issue by davidscotson** [mrjones2014/tldr.nvim#2]
On Mac OS X, with tealdeer and most other things installed via brew, I didn't get color output.

I edited  tldr-pager.sh and used
`eval "$TLDR_COMMAND --color=always $TLDR_QUERY | less -R"`

which seemed to work, though not sure if it's the correct way to do it.

**Comment by mrjones2014** [comment]
We'd have to check whether `--color` is a valid argument for all the clients. For example, the reason I'm doing `| less -R` in the script is because, while tealdeer has a built-in pager argument, the node client does not, so I'm piping to `less` as the pager.

I can check this out, and as long as the `--color` argument is valid for all clients, I'll implement your suggestion.

Thanks for filing an issue!

**Comment by mrjones2014** [comment]
Looks like the Node client does not have a `--color` option available, [despite their README.md claiming it does](https://github.com/tldr-pages/tldr-node-client#colors-under-cygwin)...

Instead, I can add a way for you to configure additional arguments to be passed via the `setup` function of the plugin.

**Comment by mrjones2014** [comment]
I've got a WIP on this branch: https://github.com/mrjones2014/tldr.nvim/tree/matjones/2-tldr-args

However it looks like this when I pass `--color=always`:

![CleanShot-2022-01-29-at-12 23 48](https://user-images.githubusercontent.com/8648891/151670767-75dcd401-6a04-47d0-ba55-3038edddb60f.png)


@davidscotson can you try with that branch and let me know if you get the same results?

**Comment by mrjones2014** [comment]
Ah, figured it out. I accidentally removed a `-R` flag to `less`.

**Comment by mrjones2014** [comment]
Fixed. You can set colors like so @davidscotson 

```lua
require('tldr').setup({ tldr_args = '--color always' })

-- or via telescope
require('telescope').setup({
  extensions = {
    tldr = { tldr_args = '--color always' }
  }
})
```
